test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/CreateTests.cs

```diff
@@ -60,6 +60,31 @@ public async Task GivenAResource_WhenPostingToHttp_TheServerShouldRespondSuccess
             DotNetAttributeValidation.Validate(observation, true);
         }
 
+        [Fact]
+        [Trait(Traits.Priority, Priority.One)]
+        public async Task GivenABinaryResource_WhenPostingToHttp_TheServerShouldRespondSuccessfully()
```

Comment:
We should include these resources where we detected errors too:

![image](https://github.com/user-attachments/assets/0b143607-d72b-456e-bbc3-7bf594f20312)

Reply:
Additional resource types added.